<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
     <class>MainWindow</class>
     <widget class="QMainWindow" name="MainWindow">
          <property name="geometry">
               <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>953</width>
                    <height>618</height>
               </rect>
          </property>
          <property name="windowTitle">
               <string>Графы</string>
          </property>
          <property name="styleSheet">
               <string notr="true"/>
          </property>
          <widget class="QWidget" name="centralwidget">
               <property name="styleSheet">
                    <string notr="true"/>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                    <item row="1" column="0">
                         <widget class="QPushButton" name="addButton_City">
                              <property name="sizePolicy">
                                   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                   </sizepolicy>
                              </property>
                              <property name="minimumSize">
                                   <size>
                                        <width>0</width>
                                        <height>30</height>
                                   </size>
                              </property>
                              <property name="styleSheet">
                                   <string notr="true"/>
                              </property>
                              <property name="text">
                                   <string>Добавить город</string>
                              </property>
                         </widget>
                    </item>
                    <item row="1" column="1">
                         <widget class="QPushButton" name="addButton_Road">
                              <property name="sizePolicy">
                                   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                   </sizepolicy>
                              </property>
                              <property name="minimumSize">
                                   <size>
                                        <width>250</width>
                                        <height>30</height>
                                   </size>
                              </property>
                              <property name="styleSheet">
                                   <string notr="true"/>
                              </property>
                              <property name="text">
                                   <string>Добавить дорогу</string>
                              </property>
                         </widget>
                    </item>
                    <item row="0" column="0" colspan="4">
                         <widget class="QGraphicsView" name="graphicsView">
                              <property name="font">
                                   <font>
                                        <family>Consolas</family>
                                   </font>
                              </property>
                              <property name="styleSheet">
                                   <string notr="true">QGraphicsView {
	border: 2px solid black;
     border-radius: 5px;
}</string>
                              </property>
                         </widget>
                    </item>
                    <item row="1" column="2">
                         <widget class="QPushButton" name="solveButton">
                              <property name="sizePolicy">
                                   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                   </sizepolicy>
                              </property>
                              <property name="minimumSize">
                                   <size>
                                        <width>0</width>
                                        <height>30</height>
                                   </size>
                              </property>
                              <property name="styleSheet">
                                   <string notr="true"/>
                              </property>
                              <property name="text">
                                   <string>Решить</string>
                              </property>
                              <property name="checkable">
                                   <bool>false</bool>
                              </property>
                              <property name="autoExclusive">
                                   <bool>false</bool>
                              </property>
                              <property name="default">
                                   <bool>false</bool>
                              </property>
                              <property name="flat">
                                   <bool>false</bool>
                              </property>
                         </widget>
                    </item>
               </layout>
          </widget>
          <widget class="QMenuBar" name="menubar">
               <property name="geometry">
                    <rect>
                         <x>0</x>
                         <y>0</y>
                         <width>953</width>
                         <height>26</height>
                    </rect>
               </property>
               <widget class="QMenu" name="menufile">
                    <property name="title">
                         <string>file</string>
                    </property>
                    <addaction name="actionsave"/>
                    <addaction name="actionload"/>
               </widget>
               <addaction name="menufile"/>
          </widget>
          <widget class="QStatusBar" name="statusbar"/>
          <action name="actionsave">
               <property name="text">
                    <string>save</string>
               </property>
          </action>
          <action name="actionload">
               <property name="text">
                    <string>load</string>
               </property>
          </action>
     </widget>
     <resources/>
     <connections/>
</ui>
